styles.css

```diff
@@ -67,12 +69,32 @@ body {
   margin-top: 40px;
   padding-right: 64px;
   padding-left: 53px;
+  border-width: 1px;
 }
 
 .dark-bg-cv {
   padding-bottom: 6px;
 }
 
+.dark-bg-submit {
+  margin-top: 10px;
+}
+
+button:hover,
+button:active {
+  background-color: #fff;
+  color: #1a2236;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 27px;
+}
+
+button:active {
+  border-width: 2px;
+}
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job making all the changes required by the previous reviewer. But I noticed that you missed some buttons to apply the same interaction for `hover`, `focused`, and `pressed` as you did for others. Those buttons are the `See Project` for each card. Make sure you make this change before you merge your pull request.